config/locales/devise/en.yml

```diff
@@ -34,9 +34,7 @@ en:
     passwords:
       choose_new_password: Choose a new password.
       invalid_token: The reset password token is invalid. Try again.
-      no_token: You can’t access this page without coming from a password reset email.
-        If you do come from a password reset email, please make sure you used the
-        full link provided.
+      no_token: In order to reset your password, please use the link provided to you in the password reset email you received. If you're pasting the link into your browser, please make sure you've pasted the entire link that was sent to you.
```

Comment:
Could we keep breaking lines at periods? Makes it a lot easier to review.

Reply:
Line breaks at periods could work...although maybe that could look awkward, too, sometimes? 

Regarding phrasing, maybe: "use the link provided to you in the password reset email that you received"?

Reply:
The main thing about breaking at periods (or commas or whatever) is keeping the line length reasonable. I try to keep that goal in mind rather than worry too much about how we get there. Periods just work well because they're natural pauses already. But whatever we can do to keep readability and review-abilty easy gets a thumbs up from me.

how about trimming down to "use the link provided in the password reset email that you received"?